shared/common-adapters/plain-input.d.ts

```diff
@@ -72,6 +72,7 @@ export type Props = {
   onBlur?: () => void
   onChangeText?: (text: string) => void
   onFocus?: () => void
+  padding?: keyof typeof globalMargins
```

Comment:
Mobile will also have to support this

Reply:
Is there something I need to fix to make sure mobile supports this?

Reply:
I _think_ mobile should support this now. lmk if I messed something up with the .native file

Reply:
tested on ios and android, everything seems to work fine